vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/ClearValueIT.java

```diff
@@ -126,6 +127,28 @@ public void allowCustomValue_enterCustomValue_setNullValue_inputElementValueIsCl
         Assert.assertEquals("", comboBox.getInputElementValue());
     }
 
+    @Test
+    public void allowCustomValue_openPopup_clearButton_selectedItemIsReset() {
+        ComboBoxElement comboBox = $(ComboBoxElement.class)
+                .id(ClearValuePage.COMBO_BOX_WITH_ALLOW_CUSTOM_VALUE_ID);
```

Comment:
Even though the test passes, it seems to be using a wrong ComboBox

https://user-images.githubusercontent.com/1222264/150092050-84331715-27ac-4449-8a17-3e725304507f.mp4

Reply:
My bad, copied the wrong test 🙈   Updated.